src/lib/Pjax/PjaxResponseMatcher.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Pjax;
+
+use EzSystems\HybridPlatformUi\Http\ResponseMatcherInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Matches a Pjax Http Response.
+ */
+interface PjaxResponseMatcher extends ResponseMatcherInterface
```

Comment:
Yeah... 'cause I wanted `ResponseMatcherInterface` to be consistent with Symfony's `RequestMatcherInterface`. Do you see my dilemma ?